stingray/io.py

```diff
@@ -346,6 +346,16 @@ def split_numbers(number, shift=0):
 
     number_F: double
         Second part of high precision number
+
+    Examples
+    --------
+    >>> n = [1234.567]
+    >>> split_numbers(n)
```

Comment:
This is not going to work. Doctests make exact comparisons and one single space is sufficient to make it fail. I suggest to compare single numbers, not arrays. And if you want to compare arrays, to just use np.all and test for True or False

Reply:
I did write a test on an array in `test_io.py`. I will change this example.

Reply:
I changed the example a little bit.